scripts/install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# create the project and add a label
+oc new-project application-monitoring
```

Comment:
Could the namespace be a param passed in or env var?
There may be other changes required though if the namespace is not `application-monitoring`

Reply:
no problem adding a param , can let it default to `application-monitoring` if not set as a param

as far as I am aware the namespace doesn't matter to the custom resources serviceMonitor and prometheusRules as they check for the  `monitoring-key: middleware`

Reply:
but the namespace is hard coded in a number of yaml so may be best to leave it for the moment 
 https://github.com/integr8ly/application-monitoring-operator/search?q=%3A+application-monitoring&unscoped_q=%3A+application-monitoring